packages/components/src/__future__/Tag/Tag.module.scss

```diff
@@ -0,0 +1,66 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/typography";
+
+.container {
+  display: inline-flex;
+  align-items: center;
+  padding: 5px 0.5rem 5px 0.5rem;
+  gap: 0.25rem;
+  border-radius: 28px;
+  font-family: $typography-paragraph-intro-lede-font-family;
+  font-size: $typography-paragraph-small-font-size;
+  font-weight: $typography-paragraph-small-font-weight;
+  line-height: $typography-paragraph-small-line-height;
+  white-space: nowrap;
+}
+
+.gray {
```

Comment:
Does the grey variant not have a different colour for the icon?

Reply:
No
https://github.com/cultureamp/kaizen-design-system/pull/3851/files#diff-35c543e702bd65369573bd34a48291a45ec1b4c5c64464f72bc5efaa63805110R43

Reply:
You actually just made me look at the Figma designs. Unless I'm looking at the wrong link, the text shouldn't even be changing between the Tags. Might be worth reaching out to @jmitsios to check.

https://www.figma.com/file/ZRfnoNUXbGZv4eVWLbF4Az/%F0%9F%96%BC%EF%B8%8F-Component-Gallery?type=design&node-id=9-37843&mode=design&t=zhd1HLgFFOVjz4yR-0

Reply:
Brought it up with the team, and this component is based on different designs
https://www.figma.com/file/hANEprZKom5Br1IBXhE8mr/%F0%9F%A7%AA-Kaizen-Labs?node-id=28588%3A183332&mode=dev

@gyfchong doesn't want to expose the link to the Figma though, so you'll want to remove the link in the docs.